src/components/button/styled.tsx

```diff
@@ -176,7 +131,7 @@ export const StyledButton = styled(({ label, icon, ...otherProps }) => (
     opacity: ${props.disabled ? 0.6 : 1.0};
     cursor: ${props.disabled ? "not-allowed" : "pointer"};
     background-color: ${buttonProps("buttonColorNormal", props)};
-    border-color: ${getColor(["green", "greenHaze"])}
+    border-color: ${getGreenHaze}
```

Comment:
Easier to change with just getColor, let's not write a special named function to get a certain color.